packages/feedback/src/core/integration.ts

```diff
@@ -234,7 +246,7 @@ export const buildFeedbackIntegration = ({
           dialog = await _loadAndRenderDialog({
             ...mergedOptions,
             onFormClose: () => {
-              dialog && dialog.close();
+              dialog && dialog.removeFromDom();
```

Comment:
IMO `onFormClose` should call `removeFromDom` just like `onFormSubmitted` since there's an `appendToDom` happening every time the button is clicked.

Reply:
wait. I think we had this as `.close()` so people could re-open the dialog and still have their in-progress message available. So like, if a user clicked outside the modal for some reason it would be closed, but not removed.

Reply:
Well that makes sense! I'll revert this bit

Reply:
Just pushed change!